packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/createDummyChangeListItems.with..st

```diff
@@ -0,0 +1,21 @@
+running
+createDummyChangeListItems: anInteger with: aBoolean
```

Comment:
I plead for naming that method `createDummyChangeListItems:withSelectionStatus:` or something like that. Until now, the selection is the only important boolean information that is stored in our `CRChangeListItem`, and even now I have to actively remember what that boolean is for.